src/data/monsters.txt

```diff
@@ -582,19 +582,19 @@ senile lihc	192	lich.gif	Atk: 56 Def: 51 HP: 50 Init: 50 P: undead E: spooky Art
 serialbus	982	serialbus.gif	Atk: 60 Def: 54 HP: 60 Init: 50 Meat: 30 P: demon E: hot Article: a	bus pass (25)
 Servant of Grodstank	799	grodseal.gif	NOCOPY FREE Atk: 175 Def: 157 HP: 200 Init: -10000 P: horror EA: stench Article: a	fustulent seal grulch (35)	infernal toilet brush (5)
 sewer snake with a sewer snake in it	1752	sewersnake.gif	Atk: 2 Def: 1 HP: 2 Init: -10000 P: beast SNAKE E: stench Article: a	carton of snake milk (100)	carton of snake milk (30)	sewer snake (30)
+shadow bat	2271	shadowbat.gif	Atk: 100 Def: 100 HP: 100 Init: 300 P: horror Phys: [min(pref(_shadowRiftCombats),90)] Elem: [min(pref(_shadowRiftCombats),90)] EA: cold EA: spooky Article: a	shadow sausage (0)
```

Comment:
Are you sure that physical resistance scales?

Here is the very first shadow monster I fought, on Feb 11:
```
place.php?whichplace=manor3&action=manor3_shadowrift
Unknown Adventure #567 = adventure.php?snarfblat=567
Unknown Adventure #567 = adventure.php?snarfblat=567
Unknown Adventure #567 = adventure.php?snarfblat=567
Unknown Adventure #567 = adventure.php?snarfblat=567
Unknown Adventure #567 = adventure.php?snarfblat=567
Unknown Adventure #567 = adventure.php?snarfblat=567

[68860] Unknown Adventure #567
Encounter: shadow devil
*** Monster 'shadow devil' has monsterId = 2273 and image 'shadowdevil.gif'
Round 0: Brianna wins initiative!
Round 1: Brianna attacks!
Round 2: shadow devil takes 1 damage.
Round 2: Brianna casts SAUCEGEYSER!
Round 3: shadow devil takes 806 damage.
```
My experience is that shadow monsters have 100% physical resistance.